packages/cli/src/models/files/ZosPackageFile.ts

```diff
@@ -27,7 +27,13 @@ export default class ZosPackageFile {
 
   constructor(fileName: string = 'zos.json') {
     this.fileName = fileName;
-    this.data = fs.parseJsonIfExists(this.fileName) || { zosversion: ZOS_VERSION };
+    try {
+      this.data = fs.parseJsonIfExists(this.fileName) || { zosversion: ZOS_VERSION };
+      // if we failed to read and parse zos.json
+    } catch(e) {
+      e.message = `Failed to read and parse ${fileName} file. Please make sure ${fileName} is present and valid. Details: ${e.message}.`;
```

Comment:
```suggestion
      e.message = `Failed to parse ${fileName} file. Please make sure that ${fileName} is a valid JSON file. Details: ${e.message}.`;
```

Reply:
Also, and given that we are allowing people to run zos from other folders, could we resolve `fileName` to the full path when outputting the error?